cbits/main.c

```diff
@@ -79,7 +79,9 @@ int hw_simd_json_sm_main(
   }
 
   uint8_t buffer[W8_BUFFER_SIZE];
-  uint32_t phi_buffer[W8_BUFFER_SIZE];
+
+  uint32_t unaligned_phi_buffer[W8_BUFFER_SIZE + 32];
+  uint32_t * phi_buffer = (uint32_t*) ((void*)unaligned_phi_buffer + ((uintptr_t)unaligned_phi_buffer & (uintptr_t)0x10));
```

Comment:
Same as the other comment.

Reply:
This is possibly overkill since `+ 32` for `uint32_t` is actually an extra `32 * 4 = 128` bytes.